fitbenchmarking/fitbenchmarking
# Adding `fitting_benchmarking` tests

#### Description of Work

Fixes #461 

Adds new tests for `fitbenchmarking/core/fitting_benchmarking.py`. This uses mock function evaluations to enable ease and speed of testing.


#### Testing Instructions

1. Check tests are clear and understandable.

Function: Does the change do what it's supposed to?

Tests: Does it pass? Is there adequate coverage for new code?

Style: Is the coding style consistent? Is anything overly confusing?

Documentation: Is there a suitable change to documentation for this change?
